.travis.yml

```diff
@@ -4,12 +4,9 @@ python:
   - "2.7"
 install:
   - "pip install funcsigs"
-  - "pip install pytest pytest-cov python-coveralls"
```

Comment:
@dorey, this line was required. See: https://travis-ci.org/kobotoolbox/formpack/builds/156954163#L566